Sources/PackageDescription/CMakeLists.txt

```diff
@@ -1,75 +1,73 @@
-        LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/pm/${PACKAGE_DESCRIPTION_VERSION}
-        RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/pm/${PACKAGE_DESCRIPTION_VERSION}
-    )
+set_target_properties(PackageDescription PROPERTIES
+    Swift_MODULE_NAME PackageDescription
+    Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/pm/ManifestAPI
+    INSTALL_NAME_DIR \\@rpath
+    OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/pm/ManifestAPI
+    OUTPUT_NAME PackageDescription
+    ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/pm/ManifestAPI
+    LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/pm/ManifestAPI
+    RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/pm/ManifestAPI
+)
 
-    if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
-      if(Foundation_FOUND)
-        target_link_libraries(PD${PACKAGE_DESCRIPTION_VERSION} PRIVATE
-          Foundation)
-      endif()
-      target_link_options(PD${PACKAGE_DESCRIPTION_VERSION} PRIVATE
-        "SHELL:-no-toolchain-stdlib-rpath")
-      set_target_properties(PD${PACKAGE_DESCRIPTION_VERSION} PROPERTIES
-          BUILD_WITH_INSTALL_RPATH TRUE
-          INSTALL_RPATH "$ORIGIN/../../$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
-    endif()
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  if(Foundation_FOUND)
+    target_link_libraries(PackageDescription PRIVATE
+      Foundation)
+  endif()
+  target_link_options(PackageDescription PRIVATE
+    "SHELL:-no-toolchain-stdlib-rpath")
+  set_target_properties(PackageDescription PROPERTIES
+    BUILD_WITH_INSTALL_RPATH TRUE
+    INSTALL_RPATH "$ORIGIN/../../$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
+endif()
 
-    if(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
-        install(FILES
-            ${CMAKE_BINARY_DIR}/pm/${PACKAGE_DESCRIPTION_VERSION}/PackageDescription.swiftinterface
-            ${CMAKE_BINARY_DIR}/pm/${PACKAGE_DESCRIPTION_VERSION}/PackageDescription.swiftdoc
-            DESTINATION lib/swift/pm/${PACKAGE_DESCRIPTION_VERSION}
-        )
-    else()
-        install(FILES
-            ${CMAKE_BINARY_DIR}/pm/${PACKAGE_DESCRIPTION_VERSION}/PackageDescription.swiftmodule
-            ${CMAKE_BINARY_DIR}/pm/${PACKAGE_DESCRIPTION_VERSION}/PackageDescription.swiftdoc
-            DESTINATION lib/swift/pm/${PACKAGE_DESCRIPTION_VERSION}
-        )
-    endif()
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
+  install(FILES
+    ${CMAKE_BINARY_DIR}/pm/ManifestAPI/PackageDescription.swiftinterface
+    ${CMAKE_BINARY_DIR}/pm/ManifestAPI/PackageDescription.swiftdoc
+    DESTINATION lib/swift/pm/ManifestAPI
+  )
+else()
+  install(FILES
+    ${CMAKE_BINARY_DIR}/pm/ManifestAPI/PackageDescription.swiftmodule
+    ${CMAKE_BINARY_DIR}/pm/ManifestAPI/PackageDescription.swiftdoc
+    DESTINATION lib/swift/pm/ManifestAPI
+  )
+endif()
 
-    install(TARGETS PD${PACKAGE_DESCRIPTION_VERSION}
-      DESTINATION lib/swift/pm/${PACKAGE_DESCRIPTION_VERSION})
-endforeach()
+install(TARGETS PackageDescription
+  DESTINATION lib/swift/pm/ManifestAPI)
```

Comment:
Rather than relying on a directory named after the version number to keep the library apart from the PackagePlugin library, I added a subdirectory here named after the subsystem for which this is an API.  The plugin one is called `PluginAPI`, and I can imagine that we'd have additional ones as needed.